server/etcdmain/config.go

```diff
@@ -186,9 +186,14 @@ func newConfig() *config {
 		"List of this member's client URLs to advertise to the public.",
 	)
 	fs.BoolVar(&cfg.ec.EnableV2Discovery, "enable-v2-discovery", cfg.ec.EnableV2Discovery, "Enable to bootstrap the cluster using v2 discovery. Will be deprecated in v3.7, and be decommissioned in v3.8.")
-	fs.StringVar(&cfg.ec.Durl, "discovery", cfg.ec.Durl, "Discovery URL used to bootstrap the cluster.")
+	fs.StringVar(&cfg.ec.Durl, "discovery", cfg.ec.Durl, "Discovery URL used to bootstrap the cluster for v2 discovery, or the cluster token for v3 discovery.")
```

Comment:
Maybe I don't understand how cluster token is used by discover, however I'm not sure about reusing v2 url for v3 cluster token. With introduction of `discovery-endpoints` V3 configuration becomes almost independent from V2, which is a good think. 

Could we maybe make the V3 configuration totally independent. For example introduce `discovery-token` to V3 configuration and mark `discover` as deprecated as it will be removed with V2?

Reply:
Then we would not need the --enable-v2-discover, as it would be driven by the other flags.

Reply:
Yes, it seems like a better solution. Thanks!

Reply:
Removed the `--enable-v2-discovery`, and added `--discovery-token`.